doc/developer_guide/index.md

```diff
@@ -78,7 +77,19 @@ pixi install
 
 The first time you run it, it will create a `pixi.lock` file with information for all available environments. This command will take a minute or so to run.
 
-All available tasks can be found by running `pixi task list`, the following sections will give a brief introduction to the most common tasks.
+All available tasks can be found by running
+
+```bash
+pixi task list
+```
+
+To list the pixi environments you have installed:
+
+```bash
+ls .pixi/envs/
+```
```

Comment:
The reason I added this info how to list the pixi envs was that I got confused which ones I had installed.

I ran into this issue ( https://github.com/prefix-dev/pixi/issues/1551#issuecomment-2281613254 ) and wasn't sure which envs I had.

I also had by accident accepted the first suggestion to create `test-310` where really I only wanted to use `test-312` consistently and avoid creating even more thatn this:
```
(base) panel $ du -hs .pixi/envs/*
968M	.pixi/envs/default
3.2G	.pixi/envs/docs
977M	.pixi/envs/lint
2.9G	.pixi/envs/test-312
1.3G	.pixi/envs/test-ui
```

Then I searched how to remove the unwanted or broken envs and found
https://pixi.sh/v0.24.2/features/environment/#cleaning-up
suggesting to just delete the folder (pixi has no command like conda to delete an env currently).

So overall I thought it could be useful to give some info here in the devdocs, but agree it's not really needed and happy to remove it if @hoxbro  you prefer to keep it short and sweet.